src/meta_memcache/base/base_cache_pool.py

```diff
@@ -47,6 +47,8 @@ def _get_pool(self, key: Key) -> ConnectionPool:
     def _encode_key(self, key: Key) -> Tuple[bytes, bool]:
         if key.is_unicode or len(key.key) > MAX_KEY_SIZE:
             return base64.b64encode(self._binary_key_encoding_fn(key)), True
+        elif " " in key.key:
```

Comment:
This will cause some problems with our usages I think - there are still some keys that should be hashed.

Reply:
I haven't seen issues in blue. But I'll keep an eye.